<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>documbuy</class>
 <widget class="QDialog" name="documbuy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_15">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Контр.агент</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Тип товару</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2_buy">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Покупка</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_sale">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Продаж</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contragent">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Тип контр.агента</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel_Order">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="locale">
    <locale language="Ukrainian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>441</width>
     <height>171</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Назва </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Кільскість</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ціна</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>None</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name_commodity_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Знижка</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_widget_commodity_movement">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Покупка/Продаж</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>168</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>Закрити</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2_add_next_commodity">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>pushButton_2_add_next_commodity</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>radioButton_2_buy</zorder>
  <zorder>radioButton_sale</zorder>
  <zorder>lineEdit_contragent</zorder>
  <zorder>label_8</zorder>
  <zorder>ErrorLabel_Order</zorder>
  <zorder>pushButton</zorder>
  <zorder>tableWidget</zorder>
  <zorder>label_17</zorder>
  <zorder>lineEdit_name_commodity_4</zorder>
  <zorder>label_18</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>comboBox_4</zorder>
  <zorder>pushButton_ok</zorder>
  <zorder>spinBox</zorder>
  <zorder>addButton</zorder>
  <zorder>label_widget_commodity_movement</zorder>
  <zorder>comboBox_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>label_17</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
